lib/setup-lib.bash

```diff
@@ -111,9 +111,11 @@ function asdf_bin_in_path() {
 
 function installed_direnv() {
   local version=$1
+  local path_without_asdf
   case "$version" in
     system | SYSTEM)
-      ASDF_DIRENV_BIN="$(run_cmd env ASDF_DIRENV_VERSION=system asdf which direnv)"
+      path_without_asdf="$(echo "$PATH" | tr ':' $'\n' | grep -v asdf | tr $'\n' ':' | sed -e 's#:$##')"
+      ASDF_DIRENV_BIN="$(run_cmd env PATH="$path_without_asdf" command -v direnv)"
```

Comment:
I don't think this is going to work quite like this? I believe `command` is a shell builtin, rather than an actual executable binary. If we wanted to run it under an altered environment, we'd probably have to do something like `bash -c "..."`. But I've got a couple of alternate ideas:

First idea: could we use `which`? on my linux and osx machine, it runs an actual binary named `/usr/bin/which`

OR, how would you feel about instead just complaining and erroring out if the shims are in the PATH? Then we could use a vanilla `command -v` here.

Reply:
You are right, trying to use `which` instead.